packages/datadog-plugin-cypress/src/plugin.js

```diff
@@ -185,8 +188,11 @@ module.exports = (on, config) => {
   let isSuitesSkippingEnabled = false
   let isCodeCoverageEnabled = false
   let testsToSkip = []
+  const unskippableSuites = []
+  let hasForcedToRunSuites = false
```

Comment:
ℹ️ This is a misnomer for cypress, because it gathers per _test_ code coverage and skips tests, but changing it here would require either updating `addIntelligentTestRunnerSpanTags` or passing it `{ hasForcedToRunSuites: hasForcedToRunTests }`, which I don't like. I'll see what I can do, but in a different PR